packages/material-ui/src/styles/createSpacing.d.ts

```diff
@@ -1,18 +1,17 @@
 /* tslint:disable:unified-signatures */
 
 export type SpacingArgument = number | string;
-export type SpacingReturnType = number | string;
 
 export interface Spacing {
-  (value1: SpacingArgument): SpacingReturnType;
-  (value1: SpacingArgument, value2: SpacingArgument): SpacingReturnType;
-  (value1: SpacingArgument, value2: SpacingArgument, value3: SpacingArgument): SpacingReturnType;
+  (value1: SpacingArgument): number;
```

Comment:
https://github.com/mui-org/material-ui/blob/a574657cb29a23594ad052fb78f972e6d3404586/packages/material-ui/src/styles/createSpacing.js#L37-L39

@oliviertassinari Were the typings wrong or the code?

Reply:
The typing I would say.